src/Markdown.MAML/Transformer/ModelTransformerBase.cs

```diff
@@ -478,5 +495,13 @@ protected string GetTextFromParagraphNode(ParagraphNode node)
         /// <param name="command"></param>
         /// <returns>true if Section was found</returns>
         protected abstract bool SectionDispatch(MamlCommand command);
+
+        protected void Report(string warning)
+        {
```

Comment:
May want to have the method like the one in MarkdownParser.cs 

void Report(string format, params object[] objects) { ... }